Components/webComponents/cartComponents/freightOptionsComponent.js

```diff
@@ -1,13 +1,18 @@
-import { FreightOptionsApi } from "./../../../API/FreightOptionsApi.js";
-import { CartApi } from "./../../../API/CartApi.js";
+import { FreightOptionsApi } from "../../../API/freightOptionsApi.js";
 
 export class FreightOptions extends HTMLElement {
     constructor() {
         super();
-
-        this.freightChoice = {
-            freight: cartState.chosenFreightOption,
-        };
+        
+        this.selectedFreightOption = null;
+        this.freightOptions = [];
+        
+        this.freightNames = {
+            PickupInStore: "Pick Up in Store",
+            Porterbuddy: "Porterbuddy",
+            PickupPoint: "Pickup Point Oslo / Sandefjord",
+            Letter: "Letter",
+        }
```

Comment:
Why is this object useful?

Reply:
it reduces code a lot from how i did it as we can access the object and get the values  from there instead of using if statements and a bunch of conditional logic.